Smartpad-iOS/Smartpad-iOS/Controller/SettingsViewController.swift

```diff
@@ -33,15 +33,28 @@ class SettingsViewController: UIViewController {
         }
         else {
             /* Show settings for paired device */
-            // TODO: Show the true paired name
-            pairedInfoLabel.text = "Paired to <paired name>"
+            let connData = ConnectionData()
+            let pairedDeviceName = connData.getSelectedPeer()
+            pairedInfoLabel.text = "Device is paired: \(pairedDeviceName)"
             unpairButton.isHidden = false
 
             changeNameLabel.text = "Changing name is not available when paired."
             changeNameField.isHidden = true
         }
     }
-
+    
+    
+    @IBAction func unpairDevice(_ sender: UIButton) {
+        ConnectionManagerAccess.connectionManager.unpairDevice()
+        pairedInfoLabel.text = "Device is not paired."
+        unpairButton.isHidden = true
```

Comment:
This code is duplicated, we just need a new function that calls the if else part inside viewDidLoad() that we call here. The connStatus will of course need to be updated